src/javascripts/app.js

```diff
@@ -1,3 +1,3 @@
 import './modules'
 
-console.log(`app.js has loaded!`)
```

Comment:
These aren't invalid, but I suppose unecessary here. Backticks are a new JS feature for doing string templating.

```js
const subject = 'world'
console.log(`hello, ${subject}!`)
// hello, world!
```